components/translation/custom_formats.rst

```diff
@@ -117,3 +117,9 @@ YAML file are dumped into a text file with the custom format::
     $dumper = new MyFormatDumper();
     $dumper->dump($catalogue, array('path' => __DIR__.'/dumps'));
 
+.. versionadded:: 2.8
+    The ability to format a message catalogue without dumping it was introduced in Symfony 2.8.
+
+.. note::
```

Comment:
Maybe moving the `versionadded` directive and the tip above the second code block in this section?